frontend/js/behaviors/core/dynamicFilterSearchBuilder.js

```diff
@@ -0,0 +1,39 @@
+const dynamicFilterSearchBuilder = function(container) {
+  const baseUrl = '/educator-resources';
```

Comment:
Hardcoding this might limit are ability to reuse this in future. (Publication's landing page doesn't use this JS behavior?) I might suggest pulling this value from a `data-*` attribute set in the container, and then hardcode a callback:

```
const baseUrl = container.getAttribute('data-base-url') || '/educator-resources';
```